packages/cli-repl/src/arg-mapper.ts

```diff
@@ -11,18 +13,33 @@ const MAPPINGS = {
   authenticationMechanism: 'authMechanism',
   keyVaultNamespace: 'autoEncryption.keyVaultNamespace',
   password: 'auth.password',
-  quiet: [ 'loggerLevel', 'error' ],
+  quiet: { opt: 'loggerLevel', val: 'error' },
   retryWrites: 'retryWrites',
   tls: 'tls',
   tlsAllowInvalidCertificates: 'tlsAllowInvalidCertificates',
   tlsAllowInvalidHostnames: 'tlsAllowInvalidHostnames',
   tlsCAFile: 'tlsCAFile',
+  tlsCRLFile: { opt: 'sslCRL', val: readCrlFileContent },
   tlsCertificateKeyFile: 'tlsCertificateKeyFile',
   tlsCertificateKeyFilePassword: 'tlsCertificateKeyFilePassword',
   username: 'auth.username',
-  verbose: [ 'loggerLevel', 'debug' ]
+  verbose: { opt: 'loggerLevel', val: 'debug' }
 };
 
+async function readCrlFileContent(crlFilePath: string): Promise<string | Buffer> {
+  let error: Error | undefined = undefined;
+  try {
+    if ((await fs.stat(crlFilePath)).isFile()) {
```

Comment:
I think you can skip this, if it’s not a readable file then `fs.readFile()` will throw anyway (there’s also some general context around this type of check being an anti-pattern – it enables [TOCTOU race conditions](https://en.wikipedia.org/wiki/Time-of-check_to_time-of-use) that can usually be avoided by not checking first, and it’s also the reason why `fs.exists()` is deprecated. But none of that really matters all that much here, I think).

Reply:
Now that you say it it totally makes sense :D And I think I kind of knew... but never heard of `TOCTOU`, thanks for the link!